modules/logging-upgrading-clo.adoc

```diff
@@ -16,6 +16,34 @@ To update the {clo} to a new major release version, you must modify the update c
 
 .Procedure
 
+. Optional: When upgrading to the {clo} 5.9 or later, you must add the `v13` schema version to the `LokiStack` custom resource before the upgrade:
```

Comment:
1) From 5.9, Lokistack support schema v13.  You can use the `v13` schema version to the `LokiStack` .

Reply:
> I can not add v13 piror 5.9

Is the v12/v13 a breaking change?
Is it okay for users to upgrade to 5.9 with v12 and without v 13?

Reply:
Yes.

Reply:
We must use schema: v13 if when logs are forward using OTLP format.  OTLP is TP feature in 6.x.

Reply:
The schema must be changed always after upgrading and it can't be done before. The reason is because the Loki version used in Logging 5.8 doesn't support the schema v13 and the good part is that the verification is avoiding to make this mistake:

```
$ oc get csv |grep -i loki
loki-operator.v5.8.15                       Loki Operator                                    5.8.15                     loki-operator.v5.8.14            Succeeded

// when tried to create in Loki 5.8 with v13
$ oc create -f logging-loki-cr.yaml 
The LokiStack "logging-loki" is invalid: spec.storage.schemas[0].version: Unsupported value: "v13": supported values: "v11", "v12"

// when tried to edit Loki 5.8 before upgrading
# lokistacks.loki.grafana.com "logging-loki" was not valid:
# * spec.storage.schemas[1].version: Unsupported value: "v13": supported values: "v11", "v12"
```

Then, the steps are:
1. Upgrade to Logging 6 or Logging 5.9
2. Alert `LokistackSchemaUpgradesRequiredp will be triggered indicating that needed to upgrade the  schema
3. Add the schema v13

Reply:
Thank you so much, both of you.